docs/assets/dev/js/editor/document/component.md

```diff
@@ -0,0 +1,62 @@
+# Legend
+* For all the examples in the __Document__ component and its subcomponents, the following legend is used, to determine which type of element is being used:
+- ![#ffd29b](https://via.placeholder.com/15/ffd29b/000000?text=+)  `Section ID` - `98c3441`
+- ![#93ff8b](https://via.placeholder.com/15/93ff8b/000000?text=+) `Column ID` &nbsp;&nbsp;- `18f2ab4`, `4b957d9`
+- ![#43f7ff](https://via.placeholder.com/15/43f7ff/000000?text=+)  `Widget ID` &nbsp;&nbsp;- `5412776`
+
+  ![legend-section-widget-tabs](_images/legend-section-widget-tabs.png)
```

Comment:
Fix the link

Reply:
![image](https://user-images.githubusercontent.com/10234691/141684187-2275c89c-ae5a-406f-b2af-ff46a67b4ae8.png)
Please check by viewing the file
https://github.com/elementor/elementor/blob/771226ac10ba50a44f2da2d189908f0cc72420a6/docs/assets/dev/js/editor/document/component.md